### PR TITLE
Linux icon fix

### PR DESCRIPTION
**Pull request by HakanSeven12**

### CODE REVIEW

File: freecad/trails/geometry/line.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def get_ortho_vector(line, distance, side=''):
     _left = Vector(-_delta.y, _delta.x, 0.0)
 
     _coord = get_coordinate(start, bearing, distance)
-
+
```
Comment (joelgraff) [inline freecad/trails/geometry/line.py:238]: Something I work really hard to do is avoid is unnecessary whitespace.  I think most code editors can auto-remove whitespace for you...  also, I haven't checked, but if you use tab to indent, make sure it converts to spaces.  Mixing spaces and tabs can create a real mess.

Comment (HakanSeven12) [reply freecad/trails/geometry/line.py:238]: I will pay attention for next time :)

File: freecad/trails/project/support/utils.py [hunk 2]
```diff
@@ -280,3 +280,16 @@ def scrub_stationing(station):
         return -1
 
     return float(scrub)
+
+def rendering_fix(base):
```
Comment (joelgraff) [inline freecad/trails/project/support/utils.py:284]: Can you explain how this works?

Comment (HakanSeven12) [reply freecad/trails/project/support/utils.py:284]: When you created a new object at origin and moved it to directly to its original position that cause shifting between object.(like my points and surfaces we talked about before). So we need to create all object at the same coordinate near to working area(like local coordinate system). To use fonction you need to send base position of your object and it return a vector. You can subtract that value from your points coordinates and create your object near to 0,0,0 than move object placement to returned vector.

Comment (joelgraff) [reply freecad/trails/project/support/utils.py:284]: I made a change to the comment.  I'll accept the function as it is, but I'm going to recommend you change the name to something more meaningful, like "compute_alternate_origin".  "rendering_fix" doesn't describe it very well.
